content/en/docs/concepts/sdk-configuration/sdk-environment-variables.md

```diff
@@ -0,0 +1,142 @@
+# OpenTelemetry Environment Variable Specification
+
+**Status**: [Mixed](document-status.md)
+
+The goal of this specification is to unify the environment variable names between different OpenTelemetry SDK implementations. SDKs MAY choose to allow configuration via the environment variables in this specification, but are not required to. If they do, they SHOULD use the names listed in this document.
+
+## Parsing empty value
+
+**Status**: [Stable](document-status.md)
```

Comment:
We tend not to add the Status within the docs here, I believe in order to avoid having to update multiple sources. 
I'm curious to see what others think about this? Could this be beneficial to readers to avoid them having to switch over to the SDK language to view the environment variable status?